docs/config/session.rst

```diff
@@ -164,6 +164,19 @@ releases.
      The session setting has no effect in CrateDB and exists for compatibility
      with ``PostgreSQL``.
 
+.. _conf-session-max_identifier_length:
+
+**max_identifier_length**
+  | *Default:* ``63``
+  | *Modifiable:* ``no``
+
+  Shows the maximum length of identifiers in bytes.
+
+  .. NOTE::
+
+     The session setting has no effect in CrateDB and exists for compatibility
```

Comment:
Should we remove this note as we move to use `MAX_INDEX_NAME_BYTES` @seut @matriv ?